### PR TITLE
Map cocina form to MODS physicalDescription

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?
See #1091

### CODE REVIEW

File: app/services/cocina/to_fedora/descriptive/form.rb [hunk 1]
```diff
@@ -17,16 +24,40 @@ def initialize(xml:, forms:)
         end
 
         def write
-          Array(forms).each_with_index do |form, _alt_rep_group|
-            write_basic(form)
-          end
+          in_physical_description = Array(forms).group_by { |form| physical_description_member?(form) }
+          Array(in_physical_description[false]).each { |form| write_basic(form) }
```
Comment (mjgiarlo) [inline app/services/cocina/to_fedora/descriptive/form.rb:28]: :eyes: what's this all about?

File: app/services/cocina/to_fedora/descriptive/form.rb [hunk 2]
```diff
@@ -17,16 +24,40 @@ def initialize(xml:, forms:)
...
+          write_physical_description(in_physical_description[true])
         end
 
         private
 
         attr_reader :xml, :forms
 
+        def physical_description_member?(form)
+          form.note.present? || ['form', 'reformatting quality', 'media type', 'extent', 'digital origin'].include?(form.type)
```
Comment (mjgiarlo) [inline app/services/cocina/to_fedora/descriptive/form.rb:37]: Could/should that array be `PHYSICAL_DESCRIPTION_TAG.keys`?

**Review by mjgiarlo — COMMENTED** [review]
Have a couple questions

File: app/services/cocina/to_fedora/descriptive/form.rb (hunk 1 above)
Comment (jcoyne) [reply app/services/cocina/to_fedora/descriptive/form.rb:28]: The form elements in cocina belong in several different places, some of them are in a node `<physicalDescription>`.  Line 27 segments the forms from cocina into those that do and those that do not belong in this node.

File: app/services/cocina/to_fedora/descriptive/form.rb (hunk 2 above)
Comment (jcoyne) [reply app/services/cocina/to_fedora/descriptive/form.rb:37]: Good idea. I'll make that change.

File: app/services/cocina/to_fedora/descriptive/form.rb (hunk 1 above)
Comment (mjgiarlo) [reply app/services/cocina/to_fedora/descriptive/form.rb:28]: OK, I see. thanks.

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**